### PR TITLE
[New Rule] Adding Lateral Movement Rules from Advanced Analytic LMD Package

### PR DESCRIPTION
**Pull request by ajosh0504**
## Summary
This PR moves the detection rules from the Lateral Movement Detection [integration](https://github.com/elastic/integrations/tree/main/packages/lmd/kibana/security_rule) package, into the detection-rules repo.

### CODE REVIEW

**Comment by ajosh0504** [comment]
@terrancedejesus @Mikaayenson FYI for when you scope out work for upcoming cycles. Also not sure why that CLA check is failing still :/

**Comment by terrancedejesus** [comment]
> @terrancedejesus @Mikaayenson FYI for when you scope out work for upcoming cycles. Also not sure why that CLA check is failing still :/

Thank you for getting this started! Yes we should try to figure out the CLA but last time we force merged it. LMD will be used for the upcoming sprint cycle.

File: rules/integrations/lmd/lateral_movement_ml_spike_in_connections_to_a_destination_ip.json
Comment (brokensound77) [inline rules/integrations/lmd/lateral_movement_ml_spike_in_connections_to_a_destination_ip.json:1]: change to `.toml`

**Comment by Mikaayenson** [comment]
> @terrancedejesus @Mikaayenson FYI for when you scope out work for upcoming cycles. Also not sure why that CLA check is failing still :/

I think the issue is your account. Let's setup time to look at it this week coming up.

File: rules/integrations/lmd/lateral_movement_ml_spike_in_connections_to_a_destination_ip.json
Comment (ajosh0504) [reply rules/integrations/lmd/lateral_movement_ml_spike_in_connections_to_a_destination_ip.json:1]: Done! Thanks for catching!

**Comment by ajosh0504** [comment]
Closing this PR. Started a [new](https://github.com/elastic/detection-rules/pull/3119) one where CLA checks are passing.

cc: @terrancedejesus @Mikaayenson

**Pull request closed without merging**